packages/cashier/src/Components/Error/form-error.jsx

```diff
@@ -44,6 +44,26 @@ const FormError = ({ disableApp, enableApp, error = {} }) => {
                     onConfirm: () => history.push(routes.proof_of_identity),
                     message: error_message,
                 });
+            } else if (error_code === 'FinancialAssessmentRequired') {
+                setDetails({
+                    title: localize('Cashier Error'),
+                    cancel_button_text: undefined,
+                    confirm_button_text: localize('OK'),
+                    onConfirm: undefined,
```

Comment:
Since this is undefined, clicking on `OK` button will not redirect to financial assessment. Is this expected?

Reply:
@farrah-deriv when the client wants to reach FA page, he click on "financial assessment" text-link which highlighted by red color. If the client wants to continue working with transfers, he clicks OK, and remains on the same page. Therefore, this behaviour is expected.